core/templates/pages/blog-dashboard-page/blog-dashboard-page.component.spec.ts

```diff
@@ -91,7 +149,7 @@ describe('Blog Dashboard Page Component', () => {
     spyOn(blogDashboardBackendApiService, 'fetchBlogDashboardDataAsync')
       .and.returnValue(Promise.resolve(blogDashboardData));
 
-    component.ngOnInit();
+    component.initMainTab();
     expect(loaderService.showLoadingScreen).toHaveBeenCalled();
     tick();
```

Comment:
Why is there a `tick()` after an `expect` stmt?

Reply:
Because we will want the `showLoadingScreen` to be called even if the promise is not resolved/rejected. Added empty lines above and below the stmt.

Reply:
Add this as a comment in the file.

Reply:
Discussed offline. Done